meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.4.0.bb

```diff
@@ -19,3 +19,6 @@ LIC_FILES_CHKSUM = " \
 # default.
 # Downprioritize this recipe in version selections.
 #DEFAULT_PREFERENCE = "-1"
+
+# Not used in this version.
+_MENDER_WRITING_TOOL_DEFAULT = ""
```

Comment:
https://northerntech.atlassian.net/browse/MEN-6938 exists to be sure we remove this from future versions.